src/prefabs/stepper.js

```diff
@@ -13,6 +13,12 @@
           key: 'activeStep',
           value: '1',
         },
+        {
```

Comment:
Maybe this could go under the advanced settings options, what do you think?

Reply:
Also I would align the label and key to be the same for both the stepper and tabs component to keep consistency.

Reply:
Don't think it should be under advanced as its probably one of the most used options.

But please rename it to "Selected design tab index" as we communicate working in the Page Builder officially as "Design mode"